### DISCUSSION TITLE
Nothing works even after 26 attempts !😥

### DISCUSSION THREAD

**Discussion by AzaleeX** [community/community#59069] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/82237910/f9fc2d98-da2b-491d-af60-29230ae8b7bf)

I send the good proof that I am a student in my high school but nothing to do I have done everything the program asks me but still no response from support I am currently on my 26th try while the documents are ok and prove that I am in this high school but nothing to do the bot which scans the images really does not want to let me pass in this program nothing to do once is that my github profile does not have the same name as written on the paper although I put my real name I do not understand I really need this high school to improve in the development in progress as at home the license opens a lot of doors I have already been admitted once without any problem but I find that it is slightly abusive to refuse 26 requests because the file is not good while it is correct with all the information requested so I would like help.

**Comment by iriteshmishra** [comment]
You could DM them the issue on [Twitter](https://twitter.com/GitHubEducation) account. They might help you over there. @AzaleeX

**Comment by samus-aran** [comment]
Hi @AzaleeX  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of your **latest** rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by AzaleeX** [reply]
> Hi @AzaleeX  👋
> 
> We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of your **latest** rejection reasons? This might help someone give you a nudge in the right direction.

Hi @samus-aran 

low he has several problems already the school id I don't have any.  The second is that I give my semester transcript.  the bot does not accept any of my images even though it has all my information and the last error is that my github profile is not complete even though it has my name, my first name, but nothing to do I still have the same  error the problem is that I had this STUDENTS plan less than 1 years ago.  The problem is that I am still in school so I should still have it but impossible to find it, so I would like my request to be made by a real person is not a robot
